src/NuGet.Core/NuGet.Packaging/PackageCreation/Utility/PackageIdValidator.cs

```diff
@@ -31,8 +31,8 @@ public static void ValidatePackageId(string packageId)
 
             if (!IsValidPackageId(packageId))
             {
-                throw new ArgumentException(String.Format(CultureInfo.CurrentCulture, NuGetResources.InvalidPackageId, packageId));
+                throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, NuGetResources.InvalidPackageId, packageId));
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
There's an empty line at the end.

Reply:
[Coding guidelines](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style), rule 18:
> Do add a new line at the end of all files.